spec/spec_helper.rb

```diff
@@ -1,13 +1,12 @@
-require 'machinist/active_record'
-require 'polyamorous/polyamorous'
+require 'ransack'
 require 'sham'
 require 'faker'
-require 'ransack'
 require 'action_controller'
 require 'ransack/helpers'
 require 'pry'
 require 'simplecov'
 require 'byebug'
+require 'machinist/active_record'
```

Comment:
`machinist/active_record` requires `active_record`, which made all of the code wrapped in `defined?(::ActiveRecord)` execute.  Requiring `machinist/active_record` after required `ransack`, I think, defines a load order that better imitates a Rails app.